fs/Cargo.toml

```diff
@@ -20,8 +20,8 @@ crate-type = ["cdylib", "rlib"]
 static_plugin = []
 
 [dependencies]
-wascc-codec = "0.9.0"
-log = "0.4.11"
+log = "0.4.14"
 env_logger = "0.7.1"
 wasmcloud-actor-blobstore = "0.1.0"
 wasmcloud-actor-core = "0.2.0"
+wasmcloud-provider-core = { path = "../../wasmcloud/crates/wasmcloud-provider-core", version = "0.1.0" }
```

Comment:
This will still unfortunately cause our actions to fail, as the `path` argument is used in local tests rather than the crates.io version. To save some PR comments I'll just leave this on this Cargo.toml, but all of these dependencies will need to be changed to:
```
wasmcloud-provider-core = "0.1.0"
```

Reply:
Having the `path` argument also affects our Makefile, as the builds for different architectures take place in a docker container which doesn't have the same access to the local filesystem